assets/src/components/propertiesPanel/ghostPropertiesPanel.tsx

```diff
@@ -38,7 +39,9 @@ const GhostPropertiesPanel = ({ selectedGhost, route }: Props) => (
       <NoWaiverBanner />
     )}
 
-    <PropertiesList vehicleOrGhost={selectedGhost} />
+    <StatusRunBlockTabs
+      statusContent={<PropertiesList vehicleOrGhost={selectedGhost} />}
```

Comment:
One thing that might help is that I was only imagining passing in content for the status tab. I only did that because this content is different for vehicles and ghosts. But I think the run and block content is all shared, so I was expecting the run and block content to live in / be nested in StatusRunBlockTabs. I think that would make it more straightforward to think about the data lifecycle.